components/WorkspacesSidebar/index.tsx

```diff
@@ -31,6 +38,11 @@ const WorkspacesSidebar: React.FC = () => {
           colorScheme="teal"
         ></IconButton>
       </HStack>
+      {workspaces.map((w) => (
+        <Link passHref href={`/workspaces/${w.id}`} key={w.id}>
+          <CLink href={`/workspaces/${w.id}`}>{w.workspaceName}</CLink>
+        </Link>
```

Comment:
Can't you just use the next link directly? Won't it retrieve the chakra styles since it's just an anchor?